src/clients/jellyfin/jellyfin.websocket.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@nestjs/common';
+import { JellyfinService } from './jellyfin.service';
+
+import { getPlaystateApi } from '@jellyfin/sdk/lib/utils/api/playstate-api';
+
+@Injectable()
+export class JellyinWebsocketService {
+  constructor(private readonly jellyfinClientManager: JellyfinService) {}
+
+  async openSocket() {
+    const systemApi = getPlaystateApi(this.jellyfinClientManager.getApi());
+
+    // TODO: Write socket playstate api to report playback progress
```

Comment:
Is this still open?

Reply:
Yes, still open. Normally, I don't merge TODO's. But in this case, I want to keep it because I keep forgetting the proper SDK files...